src/Common/test/Common.Security.Test/CertificateRotationServiceTest.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Steeltoe.Common.Options;
+using Steeltoe.Common.Utils.IO;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using Xunit;
+
+namespace Steeltoe.Common.Security.Test;
+
+public class CertificateRotationServiceTest
+{
+    [Fact]
+    [Trait("Category", "SkipOnMacOS")]
+    public async Task ServiceLoadsCertificate()
+    {
+        using var personalCertStore = new X509Store(StoreName.My, StoreLocation.CurrentUser);
+        personalCertStore.Open(OpenFlags.ReadWrite);
+        personalCertStore.Certificates.Find(X509FindType.FindByIssuerName, "Diego Instance Identity Intermediate CA", false).Should().BeEmpty();
+
+        using var sandbox = new Sandbox();
+        var filename = sandbox.CreateFile("fakeCertificate.p12");
+        File.Copy("instance.p12", filename, true);
+
+        var config = new ConfigurationBuilder()
```

Comment:
Please don't reintroduce abbreviations like "config", "cert", "certWriter" etc. in tests.

Reply:
ah yeah, this is copy/pasted... I had already fixed a few but I'll take another pass and see if I can fix the rest